main/admin-login.js

```diff
@@ -0,0 +1,61 @@
+function Adminlogin(event) {
+    
+    event.preventDefault();
+    var emailInput = document.getElementById("email");
+    var passwordInput = document.getElementById("password");
+    var emailandpassword = document.getElementById("invalid");
+    var email_invalid = document.getElementById("email_invalid");
+
+    var password_invalid = document.getElementById("password_invalid");
+
+    if (!checkPassword(passwordInput.value)) {
+        passwordInput.style.border = "solid 2px red";
+        password_invalid.style.display = "block";
+        
+    } else {
+        passwordInput.style.border = "solid 1px #cbd5e0";
+
+    }
+    if (!check_email(emailInput.value)) {
+        emailInput.style.border = "solid 1px red";
+        email_invalid.style.display = "block";
+
+    } else {
+        emailInput.style.border = "solid 1px #cbd5e0";
+
+    }
+    if (checkPassword(passwordInput.value) && check_email(emailInput.value)) {
+        passwordInput.style.border = "solid 1px #cbd5e0";
+        emailInput.style.border = "solid 1px #cbd5e0";
+        if (localStorage.getItem("user")) {
+            const user = JSON.parse(localStorage.getItem("user"));
+            if (user.email == emailInput.value && user.password == passwordInput.value) {
+                window.location.replace("http://127.0.0.1:5500/my-brand/pages/admin-articles-dashboard.html");
```

Comment:
Why do we have the links hardcoded what if I am accessing this from the server where it s deployed please don't make them statics